holmes/core/llm.py

```
@@ -403,6 +403,8 @@ def completion(
         # Get the litellm module to use (wrapped or unwrapped)
         litellm_to_use = self.tracer.wrap_llm(litellm) if self.tracer else litellm
 
+        if "ssl_verify" in self.args and self.args["ssl_verify"] is False:
+            litellm.client_session = httpx.Client(verify=False)
```

Ruff (0.14.2)

Probable use of `httpx` call with `verify=False` disabling SSL certificate checks

(S501)